src/Files.Uwp/Helpers/FilePropertiesHelpers.cs

```diff
@@ -60,65 +56,63 @@ public static async Task OpenPropertiesWindowAsync(object item, IShellPage assoc
             {
                 if (WindowDecorationsHelper.IsWindowDecorationsAllowed)
                 {
-                    AppWindow appWindow = await AppWindow.TryCreateAsync();
-
-                    Frame frame = new Frame();
+                    var frame = new Frame();
                     frame.RequestedTheme = ThemeHelper.RootTheme;
                     frame.Navigate(typeof(Properties), new PropertiesPageNavigationArguments()
                     {
                         Item = item,
                         AppInstanceArgument = associatedInstance
                     }, new SuppressNavigationTransitionInfo());
-                    ElementCompositionPreview.SetAppWindowContent(appWindow, frame);
-                    (frame.Content as Properties).appWindow = appWindow;
 
-                    appWindow.TitleBar.ExtendsContentIntoTitleBar = true;
-                    appWindow.Title = "PropertiesTitle".GetLocalized();
-                    appWindow.PersistedStateId = "Properties";
-                    WindowManagementPreview.SetPreferredMinSize(appWindow, new Size(460, 550));
+                    // Initialize window
+                    var propertiesWindow = new WinUIEx.WindowEx();
+                    var appWindow = propertiesWindow.AppWindow;
+
+                    // Set content
+                    propertiesWindow.Content = frame;
+                    if (frame.Content is Properties properties)
+                        properties.appWindow = appWindow;
+
+                    // Set min size
+                    propertiesWindow.MinWidth = 460;
+                    propertiesWindow.MinHeight = 550;
+
+                    // Set backdrop
+                    propertiesWindow.Backdrop = new WinUIEx.MicaSystemBackdrop() { DarkTintOpacity = 0.8 };
 
-                    bool windowShown = await appWindow.TryShowAsync();
-                    if (windowShown)
+                    if (AppWindowTitleBar.IsCustomizationSupported())
```

Comment:
@ahmed605 it looks like we lost your work here, any ideas on why this isn't working to hide the minimize/maximize buttons? Not sure if this is related but the properties window no longer opens up next to the mouse cursor.

Reply:
Hiding buttons apparently not supported: 
>These UWP AppWindow APIs have no direct 1:1 mapping to a Windows App SDK API.
[AppWindowTitleBar.SetPreferredVisibility](https://docs.microsoft.com/en-us/uwp/api/windows.ui.windowmanagement.appwindowtitlebar.setpreferredvisibility). 

https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/migrate-to-windows-app-sdk/guides/windowing

> Properties window no longer opens up next to the mouse cursor

Code commented out, need to be ported (I was hoping winui3 did it by default)

Reply:
We should remove the code for that, we currently have two close buttons on the properties window.